lib/src/base-styles/_nx-colors.scss

```diff
@@ -55,9 +55,11 @@
   --nx-color-alert-warning: var(--nx-swatch-orange-50);
   --nx-color-alert-error: var(--nx-swatch-red-40);
 
-  //NxBinaryDonutChart variables
-  --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-95);
-  --nx-color-binary-donut-chart-arc-stroke: var(--nx-swatch-teal-60);
+  --nx-color-progress-background: var(--nx-swatch-indigo-95);
+  --nx-color-progress-fill: var(--nx-swatch-teal-60);
```

Comment:
`NxProgressBar`, `NxMeter` and `NxBinaryDonutChart` use the two variables above for their background and fill colors. I tried coming up with a name that would describe all three components without being too specific. I went with Abhinav's suggestion: progress. Though honestly, I think there could be a better name. If there is a better name that describes all three components, let me know. I was thinking "container", "fillable-indicator" or "tracker" could also work.

Reply:
I'm not the best on naming, so others may have a better suggestion. It is tricky to get something that signifies all three, but "progress" or "tracker" do an ok job in my opinion. I think "container" is a bit confusing, and "fillable-indicator" may be odd in context `--nx-color-fillable-indicator-fill`

Reply:
> I'm not the best on naming, so others may have a better suggestion. It is tricky to get something that signifies all three, but "progress" or "tracker" do an ok job in my opinion. I think "container" is a bit confusing, and "fillable-indicator" may be odd in context `--nx-color-fillable-indicator-fill`

I think "progress" is growing me so I'll keep it as is. I'm going to close this conversation. If you have any other ideas, let me know